specification/quota/resource-manager/Microsoft.Quota/preview/2019-07-19-preview/quota.json

```diff
@@ -0,0 +1,609 @@
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "createQuotaRequests",
+            "in": "body",
+            "description": "Quota requests payload.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CreateGenericQuotaRequestParameters"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Quota requests will be created asynchronously. A request id for each changed resource is returned in the same order as the order in which they were provided.",
+            "schema": {
+              "$ref": "#/definitions/QuotaRequestResponseStatus"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "QuotaRequests"
+        ],
+        "summary": "For the specified location, gets the Quota request status by request name provided during the quota request creation for the specific resources under the subscription.",
+        "description": "Gets the QuotaRequest details and status by the quota Request Name for the resources for the resource  provider at a specific location.",
+        "operationId": "QuotaRequests_Status_Check",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ProviderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/QuotaRequestNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok. Returning the status for all relevant quota requests .",
+            "schema": {
+              "$ref": "#/definitions/QuotaRequestDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      },
+      "delete": {
+        "tags": [
+          "QuotaRequests"
+        ],
+        "summary": "Deletes the QuotaRequest by name for the resources for the resource  provider at a specific location.",
+        "description": "For the specified location, deletes the quota request by request name for resources under the subscription. Delete will not cancel the Quota Request in progress. It will only delete the completed requests, whican be rejected or succeeded quota requests.",
```

Comment:
```suggestion
        "description": "For the specified location, deletes the quota request by request name for resources under the subscription. Delete will not cancel the Quota Request in progress. It will only delete the completed requests, which can be rejected or succeeded quota requests.",
```